neo4j/datadog_checks/neo4j/__about__.py

```diff
@@ -1 +1,2 @@
-__version__ = '1.0.1'
+
+__version__ = '4.0.0'
```

Comment:
This is the version of the check not the version of neo4j, probably you can make it 1.2.0 or 2.0.0